ui/src/components/features/build-panel/components/create-assistant.tsx

```diff
@@ -109,5 +107,10 @@ export function CreateAssistant() {
     });
   }
 
-  return <AssistantForm form={form} onSubmit={onSubmit} />;
+  return (
+    <>
+      <h1 className="text-2xl font-bold">Create Assistant</h1>
```

Comment:
Not sure how I feel about the title, although I understand why you added it.. hmmm i wonder if theres another solution

Reply:
Ok I think it's way more readable / better understanding of what's going on. I may tweak the styling later - I will tag you in the PR and see what you think